src/sinks/util/adaptive_concurrency/mod.rs

```diff
@@ -29,6 +29,14 @@ pub(self) fn instant_now() -> std::time::Instant {
 #[derive(Clone, Copy, Debug)]
 #[serde(deny_unknown_fields)]
 pub struct AdaptiveConcurrencySettings {
+    /// The initial concurrency limit to use. If not specified, the initial limit will be 1 (no concurrency).
```

Comment:
Can you execute `make generate-component-docs` to update the docs as well?

Reply:
Updated!